src/components/version-context-switcher/index.tsx

```diff
@@ -25,8 +25,25 @@ const VersionContextSwitcher = ({
 	options,
 }: VersionContextSwitcherProps): ReactElement => {
 	const currentProduct = useCurrentProduct()
-	const [selectedVersion, setSelectedVersion] =
-		useState<ContextSwitcherOption['value']>(initialValue)
+	let latestVersion = options[0].value
```

Comment:
I think this is the same algorithm? Could be nice to have it all part of one assignment, but I totally understand if this is just restoring code that was there previously.

```js
const latestVersion =
  options.find(({ label }) => label.includes("(latest)"))?.value ??
  options[0].value;
```

Reply:
Yes I do like this better. It was indeed a straight copy paste of what was there before. Hopefully we can remove this again while verifying the root cause of the TF sorting issue.